src/styles/settings/_typography.scss

```diff
@@ -17,7 +17,7 @@ $weights: (
 );
 
 @mixin font( $name) {
-  @if ( map-has-key($fonts, $name) && map-has-key($weights, $name)) {
+  @if ( map-has-key($fonts, $name) & map-has-key($weights, $name)) {
```

Comment:
I am not super familiar with Sass, but it sounds like the error message was suggesting `and` instead of `&`. The [documentation](https://sass-lang.com/documentation/operators) I can find suggests that as well. If `& is correct, could you please explain? Thank you!

Reply:
Oh! I think you're right.  I saw that double ampersand was incorrect and just assumed the right answer was to use single ampersand, not `and`.